bin/restic_backup.sh

```diff
@@ -101,3 +104,30 @@ wait $!
 #wait $!
 
 echo "Backup & cleaning is done."
+
+#
+# (optionally) Notify about backup summary stats.
+#
+# How to perform the notification is up to the user; the script only writes the info to the user-owned file in a fire
+# and forget fashion.
+#
+# One option to trigger desktop notifications on user-side is using a special FIFO file (a.k.a. pipe file), which will
+# work as a queue; plus, a user process to read from that queue and run a desktop notification command.
+#
+# TODO Clean/or rephrase the example comments below
+# In this case I'm running a user process that reads from a special pipe file and sends a desktop notification using
+# `notify-send`.
+#
+# See: https://github.com/gerardbosch/dotfiles-linux/blob/main/home/.config/autostart/notification-queue.desktop and
+#      https://github.com/gerardbosch/dotfiles-linux/blob/main/home/bin/notification-queue-start-processing
```

Comment:
As I need to have this setup up on my machines as well, to be able to maintain everything that is integreated to this project, I created a setup for macOS that we can link to here

```
Examples of setup:
* Linux + notify-send
    * https://github.com/gerardbosch/dotfiles-linux/blob/main/home/bin/notification-queue-start-processing
    * https://github.com/gerardbosch/dotfiles-linux/blob/main/home/.config/autostart/notification-queue.desktop
* macOS + terminal-notifier
   * https://github.com/erikw/dotfiles/blob/personal/bin/notification-queue-notifier.sh
   * https://github.com/erikw/dotfiles/blob/personal/bin/com.user.notificationqueue.plist
```

Reply:
I added this, and changed the links with its permalink. This will reduce the chance of broken links, at the downside of not pointing the last version in case they change. But as they're placed here as example/doc I guess that having a link that "cannot" break is preferred.